.gitignore

```diff
@@ -14,3 +14,16 @@ lib
 packages/element-plus/version.ts
 /es/
 es
+
+.vitepress/*.js
```

Comment:
IMO, I think these should be moved to `docs/.gitignore`

Reply:
You were right, let's move it